fw-child/resources/scss/child.scss

```diff
@@ -441,6 +441,19 @@ $control-slider-handle-width: 2.75rem;
 
 // BREADCRUMB
 
+#header-breadcrumb {
+  overflow-x: scroll;
```

Comment:
On Chrome Windows, this CSS instruction has the effect of always showing a scrollbar.

![image](https://github.com/CanadianClimateDataPortal/climatedata-wp-theme/assets/159493515/f83cedda-6978-44b6-990b-0c1cde18ad5a)

Reply:
@xfrenette-crim this was changed to not scroll on desktop. Thanks!